include/hw_unique_key.h

```diff
@@ -20,15 +20,62 @@
 extern "C" {
 #endif
 
+#include <device.h>
+
+#if defined(CONFIG_HAS_HW_NRF_CC310)
+#define HUK_SIZE_WORDS 4
+#elif defined(CONFIG_HAS_HW_NRF_CC312)
+#define HUK_SIZE_WORDS 8
+#endif
+#define HUK_SIZE_BYTES (HUK_SIZE_WORDS * 4)
+
+/* keyslots are 1-indexed so this is the first. */
+#if defined(CONFIG_HAS_HW_NRF_CC310)
+#define HUK_KEYSLOT_KDR  1
+#endif
+#define HUK_KEYSLOT_MKEK 3
+#define HUK_KEYSLOT_MEXT 5
+
+static const uint32_t slots[] = {
```

Comment:
Can you document the entries in this static table?
Would be nice, since it is slightly confusing that you mix CC310 defines and SOC specific defines